research/conditional/train/configs/baselines/common.yaml

```diff
@@ -20,10 +21,10 @@ params:
   # shallower bass
   cutoff: 256
   batch_size: 256
-  n_steps: 275_000
+  n_steps: 250_000
   scheduler: cosine
+  learning_rate: 1e-4
   lr_warmup_steps: 2500
-  final_lr_step: 250_000
```

Comment:
revert

Reply:
why was this resolved. The final lr step is supposed to be 250K. the final training step is 275K. This line is required.